graspologic/embed/lse.py

```diff
@@ -167,12 +169,29 @@ def fit(self, graph: Union[np.ndarray, networkx.Graph], y=None):
         self._reduce_dim(L_norm)
 
         # for out-of-sample
-        inv_eigs = np.diag(1 / self.singular_values_) / (
-            self.n_features_in_ / self.n_components_
-        )
```

Comment:
why were we originally dividing by (self.n_features_in_ / self.n_components_)?

Reply:
Realized this was slightly off math wise, the current version is correct, they end up being about the same though